src/server/pfs/cmds/cmds.go

```diff
@@ -39,6 +45,8 @@ const (
 	// DefaultParallelism is the default parallelism used by get-file
 	// and put-file.
 	DefaultParallelism = 10
+
+	createPipelineTracingEnvVar = "PACH_TRACING_TARGET_REPO"
```

Comment:
Is this env-var specific to `create-pipeline` tracing? If so why is it in the `pfs/cmds` file?

Reply:
All of this stuff got moved to `src/client/pkg/tracing/extended/extended.go`

Reply:
That sounds reasonable, one more small thing, why does it end in `REPO`? I can't see what that env var has to do with repos.

Reply:
Ah, so the idea is that if you're tracing a commit/job, then you can tell the trace how far you want it to go. E.g. if your DAG is `A -> B -> C`, then you can set `PACH_TRACING_TARGET_REPO=B` and then your trace will only include events through `finishCommit(B@master)`, to keep potentially noisy stuff out. For tracing `CreatePipeline`, this is ignored.

I agree that this is a bit clunky (and it's clunky to have two environment variables in general). I'm certainly open to revising this—my sense, though, is that the design doesn't have to be solidified quite this minute